default cash in hand and cash in bank

amponce opened the pull request:
The full FSR flow should include the Cash in bank and Cash on hand pages while removing the Cash, Checking accounts, Savings accounts from the monetary-asset-values page (monetary Assets array in checkboxSelections.js since the values are being tracked with the new pages.

Since this could effect in progress forms we should determine if it's necessary to shift this update behind a feature flag. This change should help with the review navigation implementation

[Here is the updated user flow on Figma](https://www.figma.com/file/fbJtTnxh1q6Z3i1vPtvucq/Streamlined-Waiver-User-Flows?type=design&node-id=0%3A1&mode=design&t=JFOe2iUooQDEB9rg-1)

Tasks

- [x]  Determine if this change needs to hide behind a feature flag
- [x]  Update depends for 'Cash in' so the pages are included in the full FSR
- [x]  Update monetary Assets array
- [x]  Update any relevant tests

Acceptance criteria

- [x]  Cash in bank and Cash on hand questions are added to the full FSR path
- [x]  monetary-asset-values page no longer shows Cash, Checking accounts, Savings accounts as checklist items
- [x]  All tests are passing

Testing

- [x]  Confirm changes wont interfere with in progress forms

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#79884

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#81538


## Testing done

-  Cash in Bank and Cash in Hand were originally included for streamlined only users

amponce committed:
default cash in hand and cash in bank

amponce committed:
Switch local flags to false

OptionSelect reviewed (dismissed):
LGTM! 🚀

Dr-Pongo commented:
@amponce I noticed a little problem with the review nav flow (not sure if we should address it in this PR, or if we can look into it more later)

Steps to the funk:
1. I completed the full form with a single 'other income' of `11999`, and small values for the `cash` questions along with a value of `12` for `U.S. Savings Bonds`.
2. Used review navigation to edit the household income chapter to edit the other income value to something small like `11`. 
3. After that I was directed to the `cash` questions and then the streamlined transition page.
4. (issue comes in here), after the streamlined transition page I was directed to `monetaryChecklist` and `monetaryValues` before it jumped back to the final review/submit page. 

I think we should just be directed back to the review and submit page after the streamlined transition page, but I'm not sure where the disconnect is coming from

amponce committed:
adjust logic for streamlined flow

amponce commented:
> @amponce I noticed a little problem with the review nav flow (not sure if we should address it in this PR, or if we can look into it more later)
> 
> Steps to the funk:
> 
> 1. I completed the full form with a single 'other income' of `11999`, and small values for the `cash` questions along with a value of `12` for `U.S. Savings Bonds`.
> 2. Used review navigation to edit the household income chapter to edit the other income value to something small like `11`.
> 3. After that I was directed to the `cash` questions and then the streamlined transition page.
> 4. (issue comes in here), after the streamlined transition page I was directed to `monetaryChecklist` and `monetaryValues` before it jumped back to the final review/submit page.
> 
> I think we should just be directed back to the review and submit page after the streamlined transition page, but I'm not sure where the disconnect is coming from

Fixed! added some additional conditional logic to cover the edge case

Dr-Pongo reviewed (approved):
LGTM! 🚀
